file/file.go

```diff
@@ -1,27 +1,36 @@
 package file
 
 import (
-	"github.com/env0/terratag/errors"
-	"github.com/hashicorp/hcl/v2"
-	"github.com/hashicorp/hcl/v2/hclwrite"
 	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/env0/terratag/errors"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclwrite"
 )
 
-func ReplaceWithTerratagFile(path string, textContent string) {
-	taggedFilename := strings.TrimSuffix(path, filepath.Ext(path)) + ".terratag.tf"
+func ReplaceWithTerratagFile(path string, textContent string, rename bool) {
 	backupFilename := path + ".bak"
 
-	log.Print("Creating file ", taggedFilename)
-	taggedFileError := ioutil.WriteFile(taggedFilename, []byte(textContent), 0644)
-	errors.PanicOnError(taggedFileError, nil)
+	if rename {
+		taggedFilename := strings.TrimSuffix(path, filepath.Ext(path)) + ".terratag.tf"
+		log.Print("[INFO] Creating file ", taggedFilename)
+		taggedFileError := ioutil.WriteFile(taggedFilename, []byte(textContent), 0644)
+		errors.PanicOnError(taggedFileError, nil)
+	}
 
-	log.Print("Renaming original file from ", path, " to ", backupFilename)
+	log.Print("[INFO] Renaming original file from ", path, " to ", backupFilename)
 	backupFileError := os.Rename(path, backupFilename)
 	errors.PanicOnError(backupFileError, nil)
+
+	if !rename {
+		log.Print("[INFO] Creating file ", path)
+		taggedFileError := ioutil.WriteFile(path, []byte(textContent), 0644)
+		errors.PanicOnError(taggedFileError, nil)
+	}
```

Comment:
I think we can DRY it out this code; 
The log, file write function and error handling is similar with different parameters between this and line 19-22.  
While we're at it, I'd consider changing the log output to "Backing up {file} to {file}.tf.bak" to be more clear on what we're doing.